os-precheck-scripts/README.md

```diff
@@ -14,7 +14,7 @@ A script written in BASH that automates some of the tasks are described in the l
 script can be used if you are planning to do the migration yourself or use in conjunction with VPC+ 
 Cloud Migration tool.
 
-**Note:** Pre-check not required for IBM Cloud classic VSI migration.
+**Note:** Pre-check not required for IBM Cloud classic VSI migration(make sure libguestfs-tools installed before migration).
```

Comment:
discuss with @robertebe26, even if libguestfs is installed by the end user, it will not break VPC+ migration process because it will still use the qemu-img command